src/datacollector/DataCollectorMain.cs

```diff
@@ -63,7 +63,7 @@ internal DataCollectorMain(IProcessHelper processHelper, IEnvironment environmen
 
         public void Run(string[] args)
         {
-            WaitForDebuggerIfEnabled();
+            DebuggerBreakpoint.WaitForDebugger("VSTEST_DATACOLLECTOR_DEBUG");
```

Comment:
Since we are moving code... Should we move these constants to shared place as well?

Reply:
I don't think so. This is only place where it is used. No reason to "share" that.